_schemaV2.graphql

```diff
@@ -10814,6 +10814,33 @@ type PhoneNumberType {
   regionCode: String
 }
 
+type PreviewSavedSearch {
+  labels: [SearchCriteriaLabel]!
+}
+
+input PreviewSavedSearchAttributes {
+  acquireable: Boolean
+  additionalGeneIDs: [String]
+  artistIDs: [String]
+  artistNationalities: [String]
+  atAuction: Boolean
+  attributionClass: [String]
+  colors: [String]
+  dimensionRange: String
```

Comment:
**suggestion**: I think it's possible to remove this since `sizes` is used instead of `dimensionRange`

Reply:
Ahh, I wasn't sure about that. Will do, thanks!